cmd/harmony/main.go

```diff
@@ -650,6 +690,9 @@ func main() {
 	// build time.
 	os.Setenv("GODEBUG", "netdns=go")
 
+	// Get aws credentials from prompt timeout 1 second if there's no input
+	awsSettingString, _ = blsgen.Readln(1 * time.Second)
```

Comment:
why this error ignored?

Reply:
The error returned is either a timeout error  or  io.EOF when there's no input from user (when aws setting is not needed).   The returned string awsSettingString  will be "" in case of both errors and it is not needed unless LoadAwsCMKEncryptedBLSKey() is called. So I just ignore the error here.